modules/sor/sorV2/lib/pathGraph/pathGraph.ts

```diff
@@ -180,6 +188,16 @@ export class PathGraph {
         }
     }
 
+    private addPoolsAsGraphNodes(pools: BasePool[]) {
+        const validPoolTypes = ['WEIGHTED', 'COMPOSABLE_STABLE'];
```

Comment:
Do we need this limitation?

Reply:
I added this to avoid Gyro and Fx pools being added as nodes, they have no add/remove liquidity support, so it could cause an error

Reply:
But this is for v3 only, right? Will gyro and fx pools in v3 also not allow that?

Reply:
Franz is right. This whole feature is for v3 only and pools will have support for all add/remove liquidity types out of the box.

Reply:
Ok, I will need to add the pool math for these operations for theses 2 pool types to calculate limits

Reply:
Maybe add a `TODO` comment explaining that this is temporary until we start adding other pool types for v3?